AccommodationService/Infrastructure/Services/PropertyService.cs

```diff
@@ -14,9 +14,33 @@ public PropertyService(IPropertyRepository propertyRepository)
     }
 
     public async Task<Property> CreateAsync(Property property)
-    {
+    {
+        if (property.MinGuests > property.MaxGuests)
```

Comment:
This can be part of fluent validation

Reply:
1. Add the package FluentValidation
2. Create validator for property request
3. Add the validators in registry in startup.cs using `        services.AddValidatorsFromAssembly(Assembly.GetExecutingAssemly());
`

Reply:
`public class PropertyValidator : AbstractValidator<GuestRequest>
{
    public PropertyValidator()
    {
        RuleFor(x => x.MinGuests).GreaterThanOrEqualTo(0);
        RuleFor(x => x.MaxGuests).GreaterThanOrEqualTo(0); 
        RuleFor(x => x)
            .Must(x => x.MinGuests < x.MaxGuests)
            .WithMessage("MinGuests must be less than MaxGuests.");
    }
}`